src/main/java/capstone/examlab/questions/service/QuestionsServiceImpl.java

```diff
@@ -36,17 +36,11 @@ public class QuestionsServiceImpl implements QuestionsService {
     public QuestionsListDto addAIQuestionsByExamId(Long examId, List<QuestionUpdateDto> questionsUpdateListDto) {
         List<QuestionDto> questionsList = new ArrayList<>();
         for (QuestionUpdateDto questionUpdateDto : questionsUpdateListDto) {
-            String uuid = UUID.randomUUID().toString();
             String questionType = "객관식";
-            Question question = questionUpdateDto.toDocument(examId, uuid, questionType);
-            questionsRepository.save(question);
-            //생성 및 조회 검증
-            Optional<Question> createdQuestion = questionsRepository.findById(uuid);
-            if (createdQuestion.isPresent()) {
-                questionsList.add(QuestionDto.fromDocument(createdQuestion.get()));
-            } else {
-                throw new NotFoundQuestionException();
-            }
+            Question question = questionUpdateDto.toDocument(examId, questionType);
+            String uuid = questionsRepository.save(question).getId();
+            question.setId(uuid);
+            questionsList.add(QuestionDto.fromDocument(question));
```

Comment:
```suggestion
            question = questionsRepository.save(question).getId();
            
            questionsList.add(QuestionDto.fromDocument(question));
```

이렇게만 해도 충분합니다

```java
            questionsRepository.save(question).getId();
            
            questionsList.add(QuestionDto.fromDocument(question));
```
아마 이렇게 해도 충분할 수 있습니다

Reply:
그러네요 변경하겠습니다